roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\navail_version: \n' docker"
```

Comment:
What's up with the `avail_version: ` being empty there? Looks like that would return yaml like:

```yaml
---
curr_version: x.y.z
avail_version:
```